src/libraries/System.IO.FileSystem/src/Resources/Strings.resx

```diff
@@ -173,6 +173,12 @@
   <data name="Argument_InvalidAppendMode" xml:space="preserve">
     <value>Append access can be requested only in write-only mode.</value>
   </data>
+  <data name="Argument_InvalidPreallocateAccess" xml:space="preserve">
+    <value>Preallocation size can be requested only in write mode.</value>
+  </data>
+  <data name="Argument_InvalidPreallocateMode" xml:space="preserve">
+    <value>Preallocation size can be requested only for new files.</value>
+  </data>
```

Comment:
Why are these resources needed in System.IO.FileSystem?

Reply:
my mistake, it ended up here due to some eager copy-pasting.